nifi-nar-bundles/nifi-standard-services/nifi-hbase-client-service-api/src/main/java/org/apache/nifi/hbase/HBaseClientService.java

```diff
@@ -40,6 +40,18 @@
             .addValidator(new ConfigFilesValidator())
             .build();
 
+    PropertyDescriptor KERBEROS_PRINCIPAL = new PropertyDescriptor.Builder()
+            .name("Kerberos Principal")
+            .description("Principal of user writing to hbase")
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
```

Comment:
I believe using the `StandardValidators.NON_EMPTY_VALIDATOR` will cause the processor to be invalid if we want to talk to a non-secure HBase cluster, since our principal would be blank/empty.